libs/wire-api/src/Wire/API/User/Client.hs

```diff
@@ -409,6 +411,20 @@ data Client = Client
   deriving (Arbitrary) via (GenericUniform Client)
   deriving (Swagger.ToSchema) via (CustomSwagger '[FieldLabelModifier (StripPrefix "client", LowerCase)] Client)
 
+instance ToSchema Client where
+  schema =
+    object "Client" $
+      Client
+        <$> clientId .= field "id" schema
+        <*> clientType .= field "id" schema
+        <*> clientTime .= field "id" schema
+        <*> clientClass .= opt (field "id" schema)
+        <*> clientLabel .= opt (field "id" schema)
+        <*> clientCookie .= opt (field "id" schema)
+        <*> clientLocation .= opt (field "id" schema)
+        <*> clientModel .= opt (field "id" schema)
+        <*> clientCapabilities .= (field "id" schema <|> pure mempty)
```

Comment:
Field names look wrong. This also means there is no Swagger test for this.

Reply:
:facepalm: I'll fix it. Thanks for spotting it.

Reply:
Actually, there are tests, but the old JSON instances were still in place, otherwise they would have indeed caught this. Fixed now.